migration/v0.33/apply/11-prometheus-operator-crds.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+ROOT="$(readlink -f "$(dirname "${0}")/../../../")"
+
+# shellcheck source=scripts/migration/lib.sh
+source "${ROOT}/scripts/migration/lib.sh"
+
+run() {
+  case "${1:-}" in
+  execute)
+    for CLUSTER in sc wc; do
+      log_info "  - applying the prometheus-operator CRDs on $CLUSTER"
+      kubectl_do $CLUSTER apply --server-side --force-conflicts -f https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/v0.67.1/example/prometheus-operator-crd/monitoring.coreos.com_alertmanagerconfigs.yaml
```

Comment:
Could you change the URLs to point to the files in the upstream folder instead? Since we already have them we should remove the dependency to having github access

Reply:
Sure !!